include/_kernel_c.h

```diff
@@ -223,14 +223,6 @@ typedef struct _pocl_image2d_array_t { dev_image_t base; }* image2d_array_t;
 typedef struct _pocl_image1d_array_t { dev_image_t base; }* image1d_array_t;
 #endif
 
-// 3.9 needs access qualifier
-// TODO: rw images
-#ifdef CLANG_OLDER_THAN_3_9
```

Comment:
Does this remove support for older than 3.9? Please do not yet (due to the HSAIL and its LLVM 3.7 requirement).